src/Fixer/Whitespace/NoTrailingWhitespaceFixer.php

```diff
@@ -70,15 +70,6 @@ public function getDefinition()
     /**
      * {@inheritdoc}
      */
-    public function getPriority()
```

Comment:
Hm, abstract fixers' method returns the same priority (0). So by default fixers should behave as described in the comment. Or?